src/app/views/PublishesConnections.svelte

```diff
@@ -0,0 +1,144 @@
+<script lang="ts">
+  import {getRelayQuality, relaysByUrl} from "@welshman/app"
+  import {ctx} from "@welshman/lib"
+  import {AuthStatus, SocketStatus, type Connection} from "@welshman/net"
+  import {displayRelayUrl} from "@welshman/util"
+  import {quantify} from "hurdak"
+  import {onMount} from "svelte"
+  import AltColor from "src/partials/AltColor.svelte"
+  import SelectButton from "src/partials/SelectButton.svelte"
+
+  export let selected: string
+  export let activeTab: string
+
+  let selectedOptions: string[] = []
+  let connectionsStatus: {[key: string]: Map<string, Connection>} = {}
+
+  const options = [
+    "Connected",
+    "Logging in",
+    "Failed to log in",
+    "Failed to connect",
+    "Waiting to reconnect",
+    "Not connected",
+    "Unstable connection",
+  ]
```

Comment:
Make this an enum or constants since the strings are used in a bunch of places here. Also, since this is the same logic used in `RelayStatus` we should pull it out into utility functions somewhere.